pom.xml

```diff
@@ -1704,6 +1710,11 @@
                 <artifactId>nemesis-module-gcs</artifactId>
                 <version>${nemesis.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.nemesis.platform.module.avalara</groupId>
```

Comment:
Shouldn't the groupId be `io.nemesis.platform.module.tax`

Reply:
Fixed.